woocommerce/woocommerce-android
# Default store location endpoint for active plugin

Merge instructions:

1. Review this PR
2. Make sure https://github.com/woocommerce/woocommerce-android/pull/5509 is merged
2. Merge fluxc PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2224
3. Update fluxc hash in this PR
4. Remove "Do not merge" label
5. Merge this PR

Closes: #5335 


### Description

This PR updates WCAndroid so it's compatible with fluxC after introduction of `/wc_stripe/terminal/locations/store` endpoint - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2224

### Testing instructions

1. Install the newest version of the Stripe Extension plugin to your store (the version hasn't been released yet, you'll need to build it locally)
2. Make sure WCPay is not active in your store
3. Open the App
4. Open app settings
5. Tap on "In-Person Payments"
6. Tap on "Manage Card Reader"
7. Tap on the "Connect card reader" button
8. Check in the debugger that `/wc_stripe/terminal/locations/store` endpoint is invoked. You can use a flipper or a similar tool for this as well.

P.S. I have only tested this via flipper.

### Images/gif
